### PR TITLE
Added Catalin name to Readme

### PR DESCRIPTION
**Pull request by volocarucatalin**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,3 @@
-# UltimateFunCenter
\ No newline at end of file
+# UltimateFunCenter
+
+Catalin Volocaru
```
Comment (volocarucatalin) [inline README.md]: Please include github name

File: README.md [hunk 2]
```diff
@@ -3,3 +3,6 @@
 ###Saochete Valentin Andrei (Andrei)
 
 ###Alexandru Constantin Nica-Herman (Nicaleet)
+
+Catalin Volocaru
```
Comment (Nicaleet) [inline README.md:7]: Please add Heading

Comment (Nicaleet) [inline README.md]: Please Add Heading

**Review by Nicaleet — CHANGES_REQUESTED** [review]

**Review by Nicaleet — APPROVED** [review]
Thank you

**Pull request merged**